File: batches_c/batch_08_minor_logic_modification/ref.c
#include <stdio.h>

int max_of_two(int a, int b) {
    if (a > b) return a;
    return b;
}

int main() {
    printf("%d\n%d\n", max_of_two(3, 7), max_of_two(10, 5));
    return 0;
}

File: batches_csharp/batch_08_minor_logic_modification/ref.cs
using System;

class MaxOfTwo {
    static int Max(int a, int b) {
        if (a > b) return a;
        return b;
    }

    static void Main() {
        Console.WriteLine(Max(3, 7));
        Console.WriteLine(Max(10, 5));
    }
}
